File: src/api/dotnet/ArithSort.cs
﻿/*++
Copyright (c) 2012 Microsoft Corporation

Module Name:

    ArithSort.cs

Abstract:

    Z3 Managed API: Arith Sorts

Author:

    Christoph Wintersteiger (cwinter) 2012-11-23

Notes:
    
--*/

using System.Diagnostics;
using System;

namespace Microsoft.Z3
{
    /// <summary>
    /// An arithmetic sort, i.e., Int or Real.
    /// </summary>
    public class ArithSort : Sort
    {
        #region Internal
        internal ArithSort(Context ctx, IntPtr obj) : base(ctx, obj) { Debug.Assert(ctx != null); }
        #endregion
    };
}

File: src/api/java/ArithSort.java
/**
Copyright (c) 2012-2014 Microsoft Corporation
   
Module Name:

    ArithSort.java

Abstract:

Author:

    @author Christoph Wintersteiger (cwinter) 2012-03-15

Notes:
    
**/

package com.microsoft.z3;

/**
 * An arithmetic sort, i.e., Int or Real.
 **/
public class ArithSort extends Sort
{
    ArithSort(Context ctx, long obj) throws Z3Exception
    {
        super(ctx, obj);
    }
};
